README.md

```diff
@@ -56,7 +56,7 @@ Setup:
 Python:
 -------
 
-git-cinnabar will use one of `python3`, `python2.7` or `python2`, whichever has
+git-cinnabar will use one of `python3`, `python`, `python2.7` or `python2`, whichever has
```

Comment:
Thanks for the patch. Can you wrap after the comma and rewrap the following line to avoid going beyond 80 characters?